install/kubernetes/helm/OWNERS

```diff
@@ -0,0 +1,4 @@
+# No changes should be made here
+# Make all changes to istio-installer
+approvers:
+  - costinm
```

Comment:
We should probably just lock down the top level install.  The scope will not be limited to just @costinm in this directory.  Instead install/OWNERS will take priority.

I would recommend the workgroup leads (@costin, @sdake) at minimum.  We really want this directory to go completely into maintenance mode.